apps/laboratory/tests/email.spec.ts

```diff
@@ -100,9 +100,18 @@ emailTest('it should show names feature only for EVM networks', async ({ library
 emailTest('it should show loading on page refresh', async () => {
   await page.page.reload()
   await validator.expectConnectButtonLoading()
+  await page.page.waitForTimeout(5000)
```

Comment:
is this required?

Reply:
Yeah it'd need to finish loading before i can open the modal again

Reply:
But we don't know when the loading will finish right? Cannot be wait for an element to be visible instead?

Reply:
I think best way here is to add a `validator.expectConnectButtonReady` or similar that does 
```
    const connectButton = this.page.getByTestId('connect-button')
    await expect(connectButton).toContainText('Connect', {
      timeout: MAX_WAIT
    }))
``` 
That way you do not use arbitrary timeouts

Reply:
Good idea, let me add that 🙏